server/helpers/custom-validators/users.ts

```diff
@@ -65,6 +65,10 @@ function isUserBlockedValid (value: any) {
   return isBooleanValid(value)
 }
 
+function isUserAutoPlayNextVideoValid (value: any) {
+  return isBooleanValid(value)
```

Comment:
shouldn't there be a single function with `return isBooleanValid(value)` instead of those redundant functions? @Chocobozzz

Reply:
No I prefer to wrap them in these functions, because if you need this validator in different places you just use the wrapper instead of thinking "wait so what is the type of this field?". Moreover it's easier to add other validators of this field in the future (for example if it must always be true etc)